database/mssql/scripts/dbscripts/PSP_PIMS_S26_00/Build/74_DML_PIMS_CLAIM.sql

```diff
@@ -53,3 +53,12 @@ GO
 
 INSERT [dbo].[PIMS_CLAIM] ([CLAIM_ID], [CLAIM_UID], [NAME], [KEYCLOAK_ROLE_ID], [DESCRIPTION], [IS_DISABLED], [CONCURRENCY_CONTROL_NUMBER], [APP_CREATE_TIMESTAMP], [APP_CREATE_USERID], [APP_CREATE_USER_GUID], [APP_CREATE_USER_DIRECTORY], [APP_LAST_UPDATE_TIMESTAMP], [APP_LAST_UPDATE_USERID], [APP_LAST_UPDATE_USER_GUID], [APP_LAST_UPDATE_USER_DIRECTORY], [DB_CREATE_TIMESTAMP], [DB_CREATE_USERID], [DB_LAST_UPDATE_TIMESTAMP], [DB_LAST_UPDATE_USERID]) VALUES (21, N'a9b9075b-2ea5-43b6-91df-2cc075453428', N'ROLE_PIMS_R', N'a9b9075b-2ea5-43b6-91df-2cc075453428', N'Read access to Property-related map layers', 0, 3, CAST(N'2021-11-29T02:17:49.903' AS DateTime), N'SEED', NULL, N'SEED', CAST(N'2021-11-29T07:42:25.990' AS DateTime), N'service', N'1b93f614-91da-4b32-b36e-bd2c6ebd12e2', N'', CAST(N'2021-11-29T02:17:49.903' AS DateTime), N'SEED', CAST(N'2021-11-29T07:42:25.967' AS DateTime), N'SEED')
 GO
+
+INSERT [dbo].[PIMS_CLAIM] ([CLAIM_ID], [CLAIM_UID], [NAME], [KEYCLOAK_ROLE_ID], [DESCRIPTION], [IS_DISABLED], [CONCURRENCY_CONTROL_NUMBER], [APP_CREATE_TIMESTAMP], [APP_CREATE_USERID], [APP_CREATE_USER_GUID], [APP_CREATE_USER_DIRECTORY], [APP_LAST_UPDATE_TIMESTAMP], [APP_LAST_UPDATE_USERID], [APP_LAST_UPDATE_USER_GUID], [APP_LAST_UPDATE_USER_DIRECTORY], [DB_CREATE_TIMESTAMP], [DB_CREATE_USERID], [DB_LAST_UPDATE_TIMESTAMP], [DB_LAST_UPDATE_USERID]) VALUES (22, N'e3dc1c05-aae5-46b8-9b80-65a492a72a0a', N'researchfile-view', N'e3dc1c05-aae5-46b8-9b80-65a492a72a0a', N'Ability view existing research files.', 0, 3, CAST(N'2022-04-12T02:17:49.903' AS DateTime), N'SEED', NULL, N'SEED', CAST(N'2022-04-12T07:42:25.690' AS DateTime), N'service', N'1b93f614-91da-4b32-b36e-bd2c6ebd12e2', N'', CAST(N'2022-04-12T02:17:49.903' AS DateTime), N'SEED', CAST(N'2022-04-12T07:42:25.670' AS DateTime), N'SEED')
```

Comment:
@dfilteau FYI it looks like every build of the claims since IS25.1 was missing the researchfile-* claims.
I've added those back in, but just wanted to let you know so that you can ensure future scripts don't remove those claims.

Reply:
I've copied your amended files into my local repo and working folders.  Should be good going forward.